src/status_im2/subs/root.cljs

```diff
@@ -193,6 +194,8 @@
 ;;intro-wizard
 (reg-root-key-sub :intro-wizard-state :intro-wizard)
 
+(reg-root-key-sub :toasts/toasts :toasts/toasts)
+(reg-root-key-sub :toasts/index :toasts/index)
```

Comment:
we decided that's better to have fewer root keys, so there should be `:toasts` key with map

Reply:
would be cool if we could add this in our guide